frontend/src/app/main/ui/workspace/tokens/modals/import.cljs

```diff
@@ -42,18 +44,102 @@
                           :type :toast
                           :level :error})))))
 
+(mf/defc import-type-dropdown*
+  {::mf/private true}
+  [{:keys [options on-click text-render default]}]
+  (let [initial-option (or (first (filter #(= (:value %) default) options))
+                           (first options))
+        selected-option* (mf/use-state initial-option)
+        selected-option @selected-option*
+        show-dropdown? (mf/use-state false)
+
+        js-options (mf/use-memo
+                    (mf/deps options)
+                    #(clj->js
```

Comment:
@hirunatan, unfortunately `mf/props` didn't work here. I tried to use it on the result of `mapv` and also on the map literal in the function body, both throws an error.

Reply:
Sorry, i meant `mf/map->props` (https://github.com/funcool/rumext?tab=readme-ov-file#mfmap-props).

But i've seen that this is not working currently. Let me think a bit more.